fitbenchmarking/core/tests/test_results_fallback.py

```diff
@@ -0,0 +1,660 @@
+"""
+Tests for renaming col tags if jacobian and hessian experience fallback
+"""
+
+import unittest
+
+from fitbenchmarking.core.results_output import (_get_all_result_tags,
+                                                 _find_non_full_columns,
+                                                 _handle_fallback_tags,
+                                                 preprocess_data)
+
+from fitbenchmarking.core.tests.test_results_output import load_mock_results
+
+
+class EdgeCaseTests(unittest.TestCase):
+    """
+    Unit tests for the edge cases.
```

Comment:
I'm not sure this is a unit test as it's testing 4 functions work together. Would integration tests be more accurate?